### PR TITLE
doc: Update known issues for Softdevice Controller

### PR DESCRIPTION
**Pull request by adrianstoica6**
ncs 1.7.0 nrf5340 issue into known_issues.rst

### CODE REVIEW

File: doc/nrf/known_issues.rst [hunk 1]
```diff
@@ -1178,6 +1178,11 @@ KRKNWK-6255: RSSI parameter adjustment is not applied
 SoftDevice Controller
 =====================
 
+.. rst-class:: v1-7-0
+
+DRGN-16113: The active scanner could assert when performing extended scanning
+  Active scanner could assert when performing extended scanning on Coded PHY with a full whitelist.
```
Comment (wiba-nordic) [inline doc/nrf/known_issues.rst:1184]:
```suggestion
DRGN-16113: Active scanner assert when performing extended scanning
  The active scanner might assert when performing extended scanning on Coded PHY with a full whitelist.
```
Is the workaround for this to ensure that the whitelist is not full?

**Review by wiba-nordic — CHANGES_REQUESTED** [review]
If the active scanner always asserts when performing extended scanning with a full whitelist, then "might assert" should instead be "asserts".

**Review by wiba-nordic — APPROVED** [review]
Looks good.

**Review by carlescufi — APPROVED** [review]

**Comment by jori-nordic** [comment]
we'll sort this out with @adrianstoica6 , please ignore the noise reviewers

**Pull request merged**